SimpleDriverComposite

rafiti12 opened the pull request:
Fixed conflict issues.

rafiti12 committed:
Merge pull request #2 from iis-powp-2022/dev-io2

Dev io2

rafiti12 committed:
SimpleDriverComposite
Added, Implemented and tested DriverComposite.

kowallus commented on src/main/java/edu/kis/powp/jobs2d/drivers/DriverComposite.java:
```diff
@@ -0,0 +1,48 @@
+        }
+    }
+
+    public List<Job2dDriver> getDriverList() {
+        return driverList;
+    }
+
+    public void setDriverList(List<Job2dDriver> driverList) {
+        this.driverList = driverList;
+    }
```
Unnecessary methods. They can allow modifying state of the composite from outside.

kowallus commented on src/main/java/edu/kis/powp/jobs2d/drivers/DriverComposite.java:
```diff
@@ -0,0 +1,48 @@
+import java.util.List;
+
+public class DriverComposite implements Job2dDriver {
+    private List<Job2dDriver> driverList;
```
Original project formatting convention used tabs (should not be changed to spaces).

kowallus reviewed (changes requested):
Two more issues.

rafiti12 committed:
DriverComposite
- deleted unnecessary methods
- changed file formatting to tabs from spaces

rafiti12 committed:
SimpleDriverComposite
- changed default constructor in DriverComposite to work properly
- created driverComposite in TestJobs2dApp using no default constructor instead
